docs/serving/configuration/config-defaults.md

```diff
@@ -30,319 +32,497 @@ data:
   allow-container-concurrency-zero: "true"
   enable-service-links: "false"
 ```
+
 See below for a description of each property.
+
 ## Properties
-### Revision Timeout Seconds
+
+### Revision timeout seconds
 {% raw %}
-revision-timeout-seconds contains the default number of
-seconds to use for the revision's per-request timeout, if
-none is specified.
+The `revision-timeout-seconds` value determines the default number of seconds to use for the revision's per-request timeout if none is specified.
 {% endraw %}
 
-**Key**: `revision-timeout-seconds`
-
-**Default**: `"300"` (5 minutes)
+* **Global key:** `revision-timeout-seconds`
+* **Per-revision annotation key:** `serving.knative.dev/revision-timeout-seconds`
+* **Possible values:** integer
+* **Default:** `"300"` (5 minutes)
 
 **Example:**
-```yaml
-apiVersion:  v1
-kind:  ConfigMap
-metadata:
-  name:  config-defaults
-  namespace:  knative-serving
-data:
-  revision-timeout-seconds: "300" # 5 minutes
-```
 
-### Max Revision Timeout Seconds
+=== "Global (ConfigMap)"
+    ```yaml
+    apiVersion:  v1
+    kind:  ConfigMap
+    metadata:
+      name:  config-defaults
+      namespace:  knative-serving
+    data:
+      revision-timeout-seconds: "300"
+    ```
+
+=== "Per Revision"
+    ```yaml
+    apiVersion: serving.knative.dev/v1
+    kind: Service
+    metadata:
+      name: helloworld-go
+      namespace: default
+    spec:
+      template:
+        metadata:
+          annotations:
+            serving.knative.dev/revision-timeout-seconds: "300"
+        spec:
+          containers:
+            - image: gcr.io/knative-samples/helloworld-go
```

Comment:
This annotation doesn't exist - the config map property configures the default for `Spec.Timeout` on the `Revision`

```suggestion
        spec:
          timeout: 300
          containers:
            - image: gcr.io/knative-samples/helloworld-go
```